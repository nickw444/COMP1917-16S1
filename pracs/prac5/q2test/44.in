44
7 1 -10 8 -5 -9 -7 -10 9 4 4 6 4 10 -2 -3 -1 8 0 7 -1 4 9 3 8 -7 -5 4 -8 -7 7 -5 0 -6 -5 -8 4 7 2 1 0 -7 -7 -3
