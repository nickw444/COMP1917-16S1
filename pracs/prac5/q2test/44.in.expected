average of array: 0.022727
